py-geopolars/src/ffi.rs

```diff
@@ -1,5 +1,5 @@
 use arrow::{array::ArrayRef, ffi};
-use polars::prelude::*;
+use polars::prelude::{ArrowField, Series};
```

Comment:
This is just a nit that I don't love imports into the global namespace. Especially for things like `ArrowField` that could have come from `arrow2`

Reply:
oh no you didn't add that! I was just explaining the diff 🙂